printer_mods/gamebofh/door-panels/README.md

```diff
@@ -0,0 +1,51 @@
+# Laser Cut Door Panels
+
+As my distaste for all solutions VHB related, I was looking for how to replace the hinges on my 2.4.  The most popular solution seems to be the [270 Degree Hinge](https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/chrisrgonzales/270_degree_hinge) which was based on [Randell's Door Hinges](https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/randell/Door_Hinges).
```

Comment:
please change these to relative links (../../chrisgonzales/270_degree_hinge for example)

Reply:
Made the changes to those two links.  Don't know what I need to do to the PR though.

In response to your earlier question, yes the hinges are a different mod, but it's these links so hopefully that answers it.